sdk/tests/wallet/transactions.rs

```diff
@@ -162,3 +162,37 @@ async fn send_nft() -> Result<()> {
 
     tear_down(storage_path)
 }
+
+#[ignore]
+#[tokio::test]
+async fn send_with_note() -> Result<()> {
+    let storage_path = "test-storage/send_with_note";
+    setup(storage_path)?;
+
+    let wallet = make_wallet(storage_path, None, None).await?;
+
+    let account_0 = &create_accounts_with_funds(&wallet, 1).await?[0];
+    let account_1 = wallet.create_account().finish().await?;
+
+    let amount = 1_000_000;
+    let tx = account_0
+        .send_amount(
+            [SendAmountParams::new(
+                *account_1.addresses().await?[0].address(),
+                amount,
+            )?],
+            Some(TransactionOptions {
+                note: Some(String::from("send_with_note")),
+                ..Default::default()
+            }),
+        )
+        .await?;
+
+    account_0
```

Comment:
retry isnt needed here, makes the test longer, but consistent i guess?

Reply:
I just copy pasted a test, I can remove it

Reply:
Done